emt4j-common/src/main/java/org/eclipse/emt4j/common/rule/ConfRuleRepository.java

```diff
@@ -127,7 +127,7 @@ public void startElement(String uri, String localName, String qName, Attributes
                 confRuleItem.setPriority(attributes.getValue("priority"));
                 List<String[]> userDefineAttrs = new ArrayList<>(attributes.getLength());
                 for (int i = 0; i < attributes.getLength(); i++) {
-                    String attrName = attributes.getLocalName(i);
+                    String attrName = attributes.getQName(i);
```

Comment:
Seems that this PR contains a merged commit.

Reply:
I thought it would be automatically merged.
Now I merged manually.   This PR is OK ?  or reOpen a pr with one commit.

Reply:
It's ok.